mc-sema/cfgToLLVM/raiseX86.h

```diff
@@ -358,7 +358,7 @@ void R_WRITE(llvm::BasicBlock *b, unsigned reg, llvm::Value *write) {
     throw TErr(__LINE__, __FILE__, "Could not find register");
 
   int regwidth = x86_64::REG_SIZE;
-  if (reg >= llvm::X86::XMM0 && reg <= llvm::X86::XMM7) {
+  if (reg >= llvm::X86::XMM0 && reg <= llvm::X86::XMM15) {
```

Comment:
1) Go up to `XMM31`.
2) Add cases for `YMM0` through `YMM31`.
3) Add cases for `ZMM0` through `ZMM31`.

A more general solution would be a register width function.